compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -641,40 +662,11 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_Op<"attention",
       if (!sumType) return std::nullopt;
       return sumType->getRank();
     }
-
-    // Method to implement for specifying output range for
-    // DestinationStyleOpInterface
-    MutableOperandRange getDpsInitsMutable() {
-      return getOutputsMutable();
-    }
-
-    SmallVector<AffineMap> getIndexingMapsArray();
-
-    AffineMap getQueryMap() {
-      return cast<AffineMap>(getIndexingMapsArray()[0]);
-    }
-    AffineMap getKeyMap() {
-      return cast<AffineMap>(getIndexingMapsArray()[1]);
-    }
-    AffineMap getValueMap() {
-      return cast<AffineMap>(getIndexingMapsArray()[2]);
-    }
-    AffineMap getOutputMap() {
-      return cast<AffineMap>(getIndexingMapsArray()[3]);
-    }
-    std::optional<AffineMap> getMaxMap() {
-      if (getNumResults() < 2)
-        return std::nullopt;
-      return cast<AffineMap>(getIndexingMapsArray()[4]);
-    }
-    std::optional<AffineMap> getSumMap() {
-      if (getNumResults() < 3)
-        return std::nullopt;
-      return cast<AffineMap>(getIndexingMapsArray()[5]);
-    }
-
-    int64_t getIterationDomainRank() {
-      return getQueryMap().getNumDims();
+    bool getTransposeV() {
```

Comment:
Can you add some comments to it? I think people would be confused because you remove the whole `transpose_v` attribute and comments above.

Reply:
done! lmk if any of the comments need more details. :)

Reply:
How about rename the function to `isTransposeV`? `get*` is more like an accessor or returning a new object to me, while here we're checking if v is transposed or not.

Also, the comment could be updated to `Returns true if the indexing map of V is a minor transpose map`. What do you think?